apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ReservedKeywords.kt

```diff
@@ -13,6 +13,10 @@ private val KOTLIN_RESERVED_WORDS = arrayOf(
     "return", "super", "this", "throw", "true", "try", "typealias", "typeof", "val", "var", "when", "while", "yield", "it", "field"
 )
 
+private val RESERVED_ENUM_VALUE_NAMES = arrayOf("name", "ordinal")
+
 fun String.escapeJavaReservedWord() = if (this in JAVA_RESERVED_WORDS) "${this}_" else this
 
-fun String.escapeKotlinReservedWord() = if (this in (JAVA_RESERVED_WORDS + KOTLIN_RESERVED_WORDS)) "${this}_" else this
\ No newline at end of file
+fun String.escapeKotlinReservedWord() = if (this in (JAVA_RESERVED_WORDS + KOTLIN_RESERVED_WORDS)) "${this}_" else this
```

Comment:
Not related to this PR but we should refactor this code to allow different escaping rules for Java vs Kotlin.

Reply:
Will be addressed in a follow-up PR